test-suite/shared/src/test/scala/org/scalajs/testsuite/junit/JUnitAssertionsTest.scala

```diff
@@ -355,4 +359,67 @@ class JUnitAssertionsTest {
     testIfAsserts(assertThrows(classOf[IndexOutOfBoundsException], ()),
         ShallNotPass)
   }
+
+  @Test def testIfAssertsTest_issue_2252(): Unit = {
+    Try(testIfAsserts(())) match {
+      case Success(_) => // As expected
+
+      case Failure(ex) =>
+        throw new AssertionError("testIfAsserts should not have thrown.").
+            initCause(ex)
```

Comment:
OK. It suggest it be left at the end of the previous line, in this case. It's too ugly like this ^^